crates/net/network/src/manager.rs

```diff
@@ -632,301 +890,84 @@ where
     type Output = ();
 
     fn poll(self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Self::Output> {
+        let start = Instant::now();
+        let mut poll_durations = NetworkManagerPollDurations::default();
+
         let this = self.get_mut();
 
-        // poll new block imports
+        // poll new block imports (dummy)
         while let Poll::Ready(outcome) = this.block_import.poll(cx) {
             this.on_block_import_result(outcome);
         }
 
-        // process incoming messages from a handle
-        loop {
-            match this.from_handle_rx.poll_next_unpin(cx) {
-                Poll::Pending => break,
-                Poll::Ready(None) => {
-                    // This is only possible if the channel was deliberately closed since we always
-                    // have an instance of `NetworkHandle`
-                    error!("Network message channel closed.");
-                    return Poll::Ready(())
+        let acc = &mut poll_durations.acc_network_handle;
+        duration_metered_exec!(
```

Comment:
do we need the macro for those two calls?

with macro this is more verbose than without

we can also get rid of of the second Instant::now if we reuse the previous network handle duration to offset the elpased

Reply:
it's favourable here to be as exact as possible with the start time

Reply:
I find these macros very hard to read, and this adds more code than without the macro.
since we're only using this twice I'd prefer using instants directly, which is just 2 lines and no additional scope

Reply:
trying to be consistent by pattern introduced in https://github.com/paradigmxyz/reth/pull/6688

Reply:
in this case, not using a macro makes it easier to read, less code and no additional scope.
and we can also save 1 syscall if we reuse the first instant.

Reply:
sure, I'm not quite sure which instant you mean?

Reply:
we can do:

```rust
let now = Instant::now();

...
poll_durations.acc_network_handle = now.elpased();


...

poll_durations.acc_swarm = now.elpased() - poll_durations.acc_network_handle;


```